theories/Makefile.coq.local

```diff
@@ -0,0 +1,15 @@
+# Local parameters for Coq's stdlib
+
+# %.vo: %.v $(PRELUDE_DEP)
+# 	$(SHOW)COQC $<
+# 	$(HIDE)$(TIMER) $(COQC) $(COQDEBUG) $(TIMING_ARG) $(COQFLAGS) $(COQLIBS) $< $(TIMING_EXTRA)
+
+PRELUDE_VO:=$(addprefix Init/, $(addsuffix .vo, Hexadecimal Number Notations Nat Byte Prelude Wf Peano Logic Logic_Type Tauto Decimal Numeral Datatypes Specif Ltac Tactics))
```

Comment:
Are you sure you want to hardcode this rather than recursively computing the dependencies of the `Init/Prelude`?

Reply:
Not sure at all, this was just a POC, I would be very happy to replace it with a cleaner approach.